static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -1,5 +1,5 @@
+import React from 'react';
```

Comment:
Do you need to import this?

Reply:
Oh, just import `{Fragment}`

I've been trying to get rid of the global react imports for the future where react might codesplit some of this stuff